src/components/checkbox/CheckboxClass.js

```diff
@@ -0,0 +1,35 @@
+import UI from '../addremove/AddRemoveClass.js';
+
+let checkboxIds = [];
+export default class Checkbox {
+  static getCheckedBoxes() {
+    const clearCompleted = document.querySelector('.clear-completed');
+    clearCompleted.onclick = () => {
+      const markedCheckbox = document.querySelectorAll('.checkbox');
+      markedCheckbox.forEach((checkbox) => {
+        if (checkbox.checked === true) {
+          const elementId = checkbox.id;
+          checkboxIds.push(elementId);
+        }
```

Comment:
- [ ] Great job implementing the clear all functionality. You are also required to have an event listener on the checkbox ( listen for a `change` event) to update the task object values to `completed: true`, which should also be persisted in `localStorage`. Upon user interaction on the checkboxes, your current state of the app doesn't mark the task as completed in `localStorage`, kindly fix that.